docs/codeql/ql-language-reference/annotations.rst

```diff
@@ -289,6 +289,38 @@ into the places where it is called. This can be useful when a predicate body is
 compute entirely, as it ensures that the predicate is evaluated with the other contextual information
 at the places where it is called.
 
+``pragma[inline_late]``
+-----------------------
+
+**Available for**: |non-member predicates|
+
+The ``pragma[inline_late]`` annotation must be used in conjunction with a
+``bindingset[...]`` pragma and together they tell the QL optimiser to inline
+the annotated predicate after join ordering and to join order callers and callee
```

Comment:
I think "callers and callee" is not clear enough. In particular, "callee" in this sentence is the annotated predicate, which should be more explicit.

How about something along these lines?

> [...]. Furthermore, the compiler uses the specified binding set for assessing join orders both inside the body of the annotated predicate and at all call sites.

Reply:
Reworded based on the suggestion.